windows-driver-docs-pr/stream/frame-server-custom-media-source.md

```diff
@@ -168,9 +182,9 @@ HKR,,CustomCaptureSourceClsid,,%CustomCaptureSource.CLSID%
 HKR,,FriendlyName,,%CustomCaptureSource.Desc%
 
 [CustomCaptureSource.ComRegistration]
```

Comment:
You need to fork this registration of the COM classes based on OS version since old OS versions do not understand device relative COM registrations for this.